prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics-release.yaml

```diff
@@ -875,13 +875,18 @@ periodics:
 
 - <<: *config-sync-ci-job
   name: release-multi-repo-7-autopilot-rapid-latest
+  decoration_config:
+     timeout: 5h
   annotations:
     testgrid-dashboards: googleoss-kpt-config-sync-release-multi-repo-7
     testgrid-tab-name: autopilot-rapid-latest
   spec:
     <<: *config-sync-ci-job-spec
     containers:
     - <<: *config-sync-ci-container
+      env:
```

Comment:
I believe you'll need to add the rest of the env vars here. Sub-fields of anchors don't get merged in the way you might expect IIRC

You can run `yq eval 'explode(.)' [file]` to verify

Reply:
Indeed it only shows the GKE_E2E_TIMEOUT when in yq output. Added other env vars.